app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -259,6 +260,11 @@ class PatientSummaryScreen :
     return effectHandlerFactory.create(this).build()
   }
 
+  override fun onAttach(context: Context) {
+    context.injector<Injector>().inject(this)
```

Comment:
I generally prefer to do this _after_ the `super.onAttach` call. The reason being that is possible that dependencies get introduced which are dependent on the attach to be completed before they get injected.

Reply:
Ah, okay. I have gotten used to this approach since it was like that in Dagger docs. But I can move it below the `super` call.

https://dagger.dev/dev-guide/android

Reply:
Yeah, let's follow a single approach in our codebase.